tests/KubernetesClient.Tests/CertUtilsTests.cs

```diff
@@ -70,15 +70,5 @@ public void LoadFromInlineDataRelativePath()
             var cert = CertUtils.GeneratePfx(cfg);
             Assert.NotNull(cert.PrivateKey);
         }
-
-        /// <summary>
-        /// Checks 
-        /// </summary>
-        [Fact]
-        public void LoadPemWithMultiCert()
```

Comment:
this test is to ensure no regression when loading bundle.

Reply:
I've updated the test to make sure the certificates we've loaded match the original certificates we meant to load. Since the implementation of _LoadPemFileCert_ has changed this test is a bit different than what you originally wrote. Let me know if there are any other preferred way to test this as the CertificateValidationTests now also include tests that verify that we can properly build certificates using the bundle.